src/os/ui/feature/tab/propertiestab.js

```diff
@@ -44,6 +44,8 @@ os.ui.Module.directive('propertiestab', [os.ui.feature.tab.propertiesTabDirectiv
  * @ngInject
  */
 os.ui.feature.tab.PropertiesTabCtrl = function($scope, $element) {
+  os.ui.feature.tab.PropertiesTabCtrl.base(this, 'constructor', $scope, $element);
```

Comment:
I didn't see a need for this to be called last, and when converted to an ES6 class you must call `super` before referencing `this`.